### DISCUSSION TITLE
how to upgrade cobbler v3.2.2 to V3.3.3 in Rockylinux8.9?

### DISCUSSION THREAD

**Discussion by wangxu186189** [cobbler/cobbler#3544] [Q&A]
how to upgrade cobbler v3.2.2 to V3.3.3 in Rockylinux8.9?

**Comment by opoplawski** [comment]
Can you give more details about what exactly you are trying to do (what packages are involved, etc) and what problems you are running into?

**Reply by wangxu186189** [reply]
I installed it in the form of an rpm package on Rockylinux 8.9, but I don't know how to upgrade to V3.3.3? Do you have a detailed operation document for updating and upgrading Cobbler? To my knowledge, currently only V3.2.2 is available for downloading the rpm package for cobler and cobler web. How to obtain the latest V3.3.3 cobler and cobler web, and what are the underlying dependency environment and Python interpreter conditions?

**Reply by SchoolGuy** [reply]
Cobbler 3.3.x doesn't ship with Cobbler Web. The new WebUI is under development still since there is a lack of resources for it. The instructions on how to upgrade to Cobbler 3.3.3 are different depending on your desired channel to obtain it.

Edit: I believe since you obtained Cobbler 3.2.2 via an RPM you want to do an upgrade of the mentioned RPM?

**Reply by opoplawski** [reply]
As @SchoolGuy noted, cobbler 3.3 dropped the web component, so I have been keeping cobbler at 3.2.X in EPEL8.  If there is strong demand for 3.3, I could provide a separate cobbler3.3 package, but I'd rather not go through the hassle.
